pyvista/utilities/reader.py

```diff
@@ -2177,7 +2177,13 @@ class TIFFReader(BaseReader):
 
     """
 
-    _class_reader = _vtk.vtkTIFFReader
+    def lazy_vtkTIFFReader(*args):
```

Comment:
We have used `staticmethod` in other lazy loaded readers to avoid the need for `*args` here.  I think it is cleaner this way.

Reply:
Check if I'm on the right path with deac1c5.

Reply:
This is looking better in the reader module, this is a clear cut case where we should not import up front.  I'll look at the more holistic changes later, we need to be careful about not letting lazy importers leak into the pyvista namespace.